src/deadline/houdini_submitter/python/deadline_cloud_for_houdini/_assets.py

```diff
@@ -279,6 +285,26 @@ def _husk_outputs(node: hou.Node) -> set[str]:
     return output_directories
 
 
+def _usd_render_outputs(node: hou.Node) -> set[str]:
```

Comment:
Is there benefit in adding a regression test for this? Depending on how the Houdini mock works it might not add anything of use, so I'm just curious what you think

Reply:
A unit test could be added. Personally I don't know how valuable it would be given how much has to be mocked and crafted to see the desired behaviour which at the end of the day would mostly just be testing does the if clause work. I would prefer this gets covered in the near future by integration tests that are actually running within Houdini and able to test these workflows e.g. USD Renders